veocean-data-farming/veocean.md

```diff
@@ -0,0 +1,111 @@
+---
+description: An overview of the governance token, veOCEAN (vote-escrowed).
+---
+# veOCEAN
+
+veOCEAN is a fork of veCRV and uses the same parameters. Nearly 500 million USD is locked across all forks of veCRV, with the leading DeFi protocols adopting this standard. By leveraging this, Ocean Protocol takes one step further towards decentralization by streaming incentives towards contributors that are aligned with delivering fundamental protocol traction.
+
+veOCEAN stands for “vote escrowed” OCEAN. By holding veOCEAN you become a governance participant, eligible to receive rewards and engage with many different protocol mechanisms.
```

Comment:
We never really explain how holding veOcean makes the user a governance participant. It only explains how they can receive rewards

Reply:
TBH there isn't really much by way of governance right now. To try to explain would add confusion.

And there's too much detail on CRV stuff. And it's a bit fluffy sounding.

Therefore, I'd replace the above two paragraphs ("veOCean is.." and "veOCEAN stands for") with something like:

"veOCEAN is a mechanism to align near-term incentives (maximize APY) with long-term incentives (long-term locking). It's a fork of veCRV contracts which have been battle-tested over years".

Reply:
Done